IsiteYves/salonsense
# Update app to English and make it more accessible

#### What does this PR do?

- Modifies to English version

#### Description of Task to be completed?

- Corrects grammar and translates relevant Kinyarwanda keywords to English

#### How should this be manually tested?

- Clone the repo

- checkout to branch `update-brand`

- Run `npm start local` in the root folder of this project

- Check the app at `https://localhost:3000` in your web browser
